### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by thienquang199x** [xmengli999/H-DenseUNet#22]
What is "0.78,0.65,8.57" in concat? Thanks

**Comment by xmengli999** [comment]
weight for background, liver, and tumor.

**Comment by thienquang199x** [comment]
How to calculate it ? Thanks

**Comment by xmengli999** [comment]
Call the dataloader for many iterations, then check the voxel number of the background, liver and tumor respectively.

**Comment by xmengli999** [comment]
Call the dataloader for many iterations, then check the voxel number of the
background, liver and tumor respectively.

**Comment by thienquang199x** [comment]
Thank you very much!

**Comment by thienquang199x** [comment]
Why weight of tumor great than liver and background ?

**Comment by xmengli999** [comment]
When you call dataloader for many iterations, you can find the voxel amount of tumor is small.

**Comment by thienquang199x** [comment]
How to calculate weight ? Tumor/sum(background,liver,tumor) ?

**Comment by xmengli999** [comment]
Yes, the weight is calculated by Tumor/sum(background,liver,tumor) during the training stages.
